source/sourcing_and_processing_data/processors/normalize_projection_reference.rst

```diff
@@ -0,0 +1,30 @@
+Normalize Projection Reference processor
+========================================
+
+This processor handles a geopoint with a projection system different from a `WGS84 <http://en.wikipedia.org/wiki/WGS_84>`_ field. The field value containing a geopoing other than a WGS84 representation is replaced with its WGS84 representation.
+
+For example, if the EPSG code is set to ``27572``, the processor will consider that the original geo field contains coordinates expressed in `Lambert Zone II <http://spatialreference.org/ref/epsg/ntf-paris-lambert-zone-ii/>`_.
+
+.. note::
+    The input must be expressed with the same logic as a WGS84 geo coordinate: ``Y,X``.
+
+Setting the processor
+---------------------
+
+To set the parameters of the Normalize Projection Reference processor, follow the indications from the table below.
+
+.. list-table::
+  :header-rows: 1
+
+  * * Label
+    * Description
+    * Mandatory
+    * Default value
+  * * Field
+    * Field that will be normalized, containing a geopoint with another projection system than WGS84
+    * yes
+    *
+  * * Source epsg code
+    * `EPSG <http://spatialreference.org/ref/epsg/>`_ code of the source coordinates system
+    * yes
+    * 4326
```

Comment:
We can remove the 'Default value' column or at least the 4326 value, it refers to the WGS84 value so it's useless (we could change the code too)